### PR TITLE
Automatically build the masking set

### PR DESCRIPTION
**Pull request by smobs**
Take advantage of the newly released functionality

### CODE REVIEW

**Comment by smobs** [comment]
Good point, I'll look at the tests.

**Comment by smobs** [comment]
I've created a PR for the tests https://github.com/red-gate/SqlDataCatalog/pull/3587.

This pr should be merged first, the old test should still pass.

File: SqlDataCatalog-SqlDataMasker/Generate masking set from catalog.ps1 [hunk 1]
```diff
@@ -15,9 +15,13 @@ Connect-SqlDataCatalog -AuthToken $authToken -ServerUrl $serverUrl
...
 
-# a masking set file that will be modified by this script, you should generate it using Redgate Data Masker.
-$inputMaskingSetPath = "AdventureWorks.DMSMaskSet"
-$outputMaskingSetPath = "AdventureWorks Generated.DMSMaskSet"
+# a masking set file that will be created by this script.
+$maskingSetPath = "AdventureWorks.DMSMaskSet"
+
+
+# Generate the masking file.  Requires Data masker version >= 7.0.17
+& 'C:\Program Files\Red Gate\Data Masker for SQL Server 7\DataMaskerCmdLine.exe' build using-windows-auth -m $maskingSetPath -l C:\Scripts\Logs -i $instanceName -d $databaseName
```
Comment (Praveen-Redgate) [inline SqlDataCatalog-SqlDataMasker/Generate masking set from catalog.ps1:23]: Could check to see if DM is installed, and warn users if DM is not installed.

**Comment by smobs** [comment]
^ @Praveen-Redgate like this?

File: SqlDataCatalog-SqlDataMasker/Generate masking set from catalog.ps1 [hunk 2]
```diff
@@ -15,9 +15,16 @@ Connect-SqlDataCatalog -AuthToken $authToken -ServerUrl $serverUrl
...
 
-# a masking set file that will be modified by this script, you should generate it using Redgate Data Masker.
-$inputMaskingSetPath = "AdventureWorks.DMSMaskSet"
-$outputMaskingSetPath = "AdventureWorks Generated.DMSMaskSet"
+# a masking set file that will be created by this script.
+$maskingSetPath = "AdventureWorks.DMSMaskSet"
+
+
+# Generate the masking file.  Requires Data masker version >= 7.0.17
+if (-not (Test-Path -Path "C:\Program Files\Red Gate\Data Masker for SQL Server 7\DataMaskerCmdLine.exe")) {
```
Comment (Praveen-Redgate) [inline SqlDataCatalog-SqlDataMasker/Generate masking set from catalog.ps1:23]:
```suggestion
if (-not (Test-Path -Path "C:\Program Files\Red Gate\Data Masker for SQL Server 7\DataMaskerCmdLine.exe" -PathType leaf)) {
```

**Comment by Praveen-Redgate** [comment]
> ^ @Praveen-Redgate like this?

Yes

**Comment by smobs** [comment]
Thanks Praveen

**Pull request merged**